.github/workflows/contracts.yml

```diff
@@ -53,37 +43,43 @@ jobs:
       - name: Install rust
         uses: actions-rs/toolchain@v1
         with:
+          profile: minimal
           default: true
-          toolchain: "${{ inputs.rust-toolchain }}"
+          toolchain: ${{ inputs.rust-toolchain }}
+          target: wasm32-unknown-unknown
 
       - name: Setup the PATH variable
-        run: echo "PATH=$HOME/.local/bin:$HOME/elrondsdk/vmtools:$PATH" >> $GITHUB_ENV
+        run: echo "PATH=$HOME/.local/bin:$HOME/multiversx-sdk/vmtools:$HOME/multiversx-sdk/nodejs/latest/bin:$PATH" >> $GITHUB_ENV
 
       - name: Install prerequisites
         run: |
-          pip3 install ${{ inputs.pip-erdpy-args }}
-          mkdir $HOME/elrondsdk
-          erdpy deps install vmtools --tag ${{ inputs.vmtools-version }}
+          pip3 install ${{ inputs.pip-mxpy-args }}
+          mkdir $HOME/multiversx-sdk
+          mxpy deps install vmtools --tag ${{ inputs.vmtools-version }}
 
-          erdpy deps install nodejs
-          erdpy deps install wasm-opt
+          mxpy deps install nodejs
+          mxpy deps install wasm-opt
 
           cargo install twiggy
 
+          cargo install multiversx-sc-meta
+
       - name: Install libtinfo5
         if: inputs.install-libtinfo5
         run: |
           sudo apt update
           sudo apt install -y libtinfo5
 
       - name: Build the wasm contracts
-        run: erdpy contract build -r ${{ inputs.extra-build-args }}
+        run: sc-meta all build --no-imports --target-dir $(pwd)/target --path .
```

Comment:
I am launching another version of `sc-meta` that has an additional flag, `--ignore`, after that we should add `--ignore target`, because it currently finds fake contract crates under `target`

Reply:
Wouldn't it be better if `sc-meta` automatically ignores `target`?
Not having to add `--ignore target` would also help when manually calling it from the CLI.

Reply:
Yes, also thinking about that. Thinking about having it by default.